src/Plugin/SearchAdapter/Aggregation/BuilderPlugin.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * ScandiPWA_CatalogGraphQl
+ *
+ * @category    ScandiPWA
+ * @package     ScandiPWA_CatalogGraphQl
+ * @author      Alfreds Genkins <info@scandipwa.com>
+ * @copyright   Copyright (c) 2018 Scandiweb, Ltd (https://scandiweb.com)
+ */
+
+declare(strict_types=1);
+
+namespace ScandiPWA\CatalogGraphQl\Plugin\SearchAdapter\Aggregation;
+
+use Magento\Framework\Search\RequestInterface;
+use Magento\Framework\Search\Request\QueryInterface;
+use Magento\Elasticsearch\SearchAdapter\Aggregation\Builder;
+
+class BuilderPlugin {
+
+     protected const PRICE_BUCKET = 'price_bucket';
```

Comment:
Same constant already exist in Model/Layer/Filter/Price.php
I guess it has not correct access modifier and can be reused here.